sktime/classification/distance_based/_time_series_neighbors.py

```diff
@@ -88,6 +63,28 @@ class KNeighborsTimeSeriesClassifier(_KNeighborsClassifier, BaseClassifier):
         - [callable] : a user-defined function which accepts an
           array of distances, and returns an array of the same shape
           containing the weights.
+    algorithm : str, optional. default = 'brute'
+        search method for neighbours
+        one of {'auto’, 'ball_tree', 'kd_tree', 'brute'}
+    distance : str or callable, optional. default ='dtw'
+        distance measure between time series
+        if str, must be one of the following strings:
+            'euclidean', 'squared', 'dtw', 'ddtw', 'wdtw', 'wddtw',
+            'lcss', 'edr', 'erp', 'msm'
```

Comment:
- the older version also had something called `twe`. Was it removed intentionally or accidentally?
- the `squared` option is not in the `DISTANCES_SUPPORTED` variable

Reply:
Sadly, there is no ground truth for `DISTANCES_SUPPORTED` in the `distances` module, so I suppose it just has to be updated permanently.

Perhaps we should open an issue.

Reply:
here: https://github.com/alan-turing-institute/sktime/issues/3080

Reply:
good idea with the issue.